harness/python/feature.py

```diff
@@ -121,3 +126,38 @@ async def check_result(self, handle: WorkflowHandle) -> None:
                     raise TypeError("Unexpected activity error") from err
             else:
                 raise err
+
+    async def get_history_events(self, handle: WorkflowHandle) -> list[HistoryEvent]:
```

Comment:
`list` is only available in 3.9+, ref https://peps.python.org/pep-0585/ (but if we want to limit to a certain Python version for features, that'd be fine)

Reply:
True, I saw that we only run the harness with python 3.10 so assumed it was fine to use.
Would you like to me use `typing.List` instead?

Reply:
> Would you like to me use typing.List instead?

Yeah, that's what I do in the Python SDK. But, I'm ok with 3.10+ Python only here. I have put a really high minimum Java version so I can use all those new fancy features too.